rocket_sqlite_oracle/src/lib.rs

```diff
@@ -0,0 +1 @@
+pub mod kraken;
```

Comment:
I don't think `rocket_sqlite_oracle` needs to have a library API. This module should be declared in `main.rs`.